src/components/TagInput/TagInput.js

```diff
@@ -103,21 +103,24 @@ export default class TagInput extends Component {
   }
 
   handleChange = debounce(value => {
-    const strippedValue = this.props.stripInputHashtag ? value.replace(/^#/, '') : value
+    let strippedValue = value
+    if (value) {
+      strippedValue = this.props.stripInputHashtag ? value.replace(/^#/, '') : value
+    }
     this.props.handleInputChange(strippedValue)
   }, 200)
 
   render () {
     let { tags, placeholder } = this.props
 
-    const { suggestions, className, theme, readOnly, maxTags, addLeadingHashtag, renderSuggestion } = this.props
+    const { suggestions, className, theme, readOnly, maxTags, addLeadingHashtag, renderSuggestion, tagType } = this.props
     if (!tags) tags = []
     if (!placeholder) placeholder = 'Type...'
 
     const optionalHashtag = addLeadingHashtag ? '#' : ''
 
     const selectedItems = uniqBy('id', tags).map(t =>
-      <li key={t.id} className={theme.selectedTag}>
+      <li key={t.id} className={theme.selectedTag} styleName='list-item-selected'>
```

Comment:
confusing to use both className and styleName, why is that needed? we already have a class on it called selectedTag. we could use the classNames library if we need two classes on it, one default and one from the theme. the different is that styleName looks up the className in whatever css file has been imported, className you have to specific the variable for the imported styles, otherwise they are the same

Reply:
agreed, using classNames library is better 👍